README.md

```diff
@@ -1 +1,33 @@
-# IPLive
\ No newline at end of file
+# Cric-Live-Scoreboard
+
+> - It is made using ReactJs 
+> - API used => [cric-api](https://www.cricapi.com/)
+> - app deployed at [cric-live](https://cric-liv.web.app/)
+
+## Installation 
+
```

Comment:
Add a forking step as well before the cloning process.

Reply:
Okay!